adding_species_manuscript.md

```diff
@@ -25,104 +25,87 @@ preamble: |
   \doublespacing
 ---
 
-# Outline
-
-1.  Overall goals and structure of stdpopsim
-
-    -   stdpopsim is a community-maintained resource intended to provide easy access to simulation frameworks
-
-        -   coding population genetic simulation models can be arduous and error-prone, so stdpopsim provides a way to:
-
-            -   avoid re-making common models
-            -   provide standard benchmarks for methods development and testing
-
-        -   and a resource for empirical researchers, eg. power analyses or sanity checks (Adrion et al. 2020)
-
-    -   feedback from 2020/2021 workshops made it clear that the prospective community included empiricists who are especially concerned with using stdpopsim for inference for individual species that mostly not already in stdpopsim catalog
-
-2.  **ELISE** Community-based expansion of the number and variety of species and their demographic scenarios included in the stdpopsim catalog is useful for both of these major goals shared by the population genetics community, methods development and inference 
-
-    -   when first published, the stdpopsim catalog included 6 species: *Homo sapiens*, *Pongo abelii*, *Canis familiaris*, *Drosophila melanogaster*, *Arabidopsis thaliana*, and *Escherichia coli.*
-
+Outline
+
+# Introduction
+1.  Simulation is important for population genomics, both methods development and inference
+    - there is a building avalanche of genomic data for a variety of species, and a complementary flourishing of methods for population genomic inference
+        - so the number and variety of species requiring detailed and accurate simulation models is increasing quickly
+    -   coding population genetic simulation models can be arduous and error-prone, but important
+        -   provide training data for machine learning, ABC
+        -   for both methods development and empirical research
```

Comment:
```suggestion
        -   for both methods development and empirical research
        - "accurate" includes realistic demographic models and genetic maps as both can strongly impact the patterns of variation and haplotype/linkage
```

Reply:
I think we want to make super clear what's important about whole-genome sims (and when it's not as important) - we'll have some discussion below, but we don't want to just say "more realism is better"/

Reply:
Absolutely. I think it's a good idea to highlight it in the intro, with more detail later. Is this something you could pull into section 3 "Making a popgen simulation" after @izabelcavassim's part on the necessary resources?